src/components/Button/Button.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Link from 'gatsby-link';
 
 const Button = ({ division, children, link }) => {
-  const location = link.substr(0, 4) == 'http'
-    ? <a href={link}>{children}</a>
-    : <Link to={link}>{children}</Link>
-  return <div className={`button ${division}`}>{location}</div>
+  const isExternalLink = link.substr(0, 3) === 'http';
+  // const location = link.substr(0, 4) == 'http'
+  //   ? <a href={link}>{children}</a>
+  //   : <Link to={link}>{children}</Link>
+  return ( 
+   //<div className={`button ${division}`}>{isExternalLink}</div> 
```

Comment:
@kylerowland You should remove this commented code before you make a commit. It's useful during development, but we should keep it clean when committing to the code base. If we ever need to see the old code, we can click the history tab for this file.

Reply:
The images were not changed and i didnt change any sidebar code so if it does conflict, its because my sidebar is old

Reply:
@kylerowland You should be able to resolve the sidebar code issues by merging master into your branch. Possibly the images too. I'm concerned your branch may be trying to change so many files because it's trying to write over new code with old code. This shouldn't happen, but if you've formatted a file (which can happen just by opening it and saving it), it will count as newer.